src/components/LandingAnimation/LandingAnimation.js

```diff
@@ -137,36 +134,37 @@ const LandingAnimation = () => {
       <StartButton
         ref={animationButtonRef}
         onClick={handelAnimationButtonClick}
+        isButtonInPlace={isButtonInPlace}
       >
         <FormattedMessage id={animationButtonTextId} />
       </StartButton>
-      <StyledSpan ref={hiSpanRef}>
-        <FormattedMessage id="animation.hiSpan" />
-      </StyledSpan>
-      <StyledSpan ref={mySpanRef}>
-        <FormattedMessage id="animation.mySpan" />
-      </StyledSpan>
-      <StyledSpan ref={hiMySpanRef}>
-        <FormattedMessage id="animation.hiMySpan" />
-      </StyledSpan>
-      <StyledSpan ref={nameSpanRef}>
-        <FormattedMessage id="animation.nameSpan" />
-      </StyledSpan>
-      <StyledSpan ref={hiMyNameSpanRef}>
-        <FormattedMessage id="animation.hiMyNameSpan" />
-      </StyledSpan>
-      <StyledSpan ref={isSpanRef}>
-        <FormattedMessage id="animation.isSpan" />
-      </StyledSpan>
-      <StyledSpan ref={hiMyNameIsSpanRef}>
-        <FormattedMessage id="animation.hiMyNameIsSpan" />
-      </StyledSpan>
-      <StyledSpan ref={karolSpanRef}>
-        <FormattedMessage id="animation.karolSpan" />
-      </StyledSpan>
-      <StyledSpan ref={hiMyNameIsKarolSpanRef}>
-        <FormattedMessage id="animation.hiMyNameIsKarolSpan" />
-      </StyledSpan>
+      <StyledParagraph ref={firstParagraphRef}>
```

Comment:
also here - maybe like `StyledMessage`? Then you can change the tag without changing the name